frontend/src/components/ScheduledEventsTableRow/ScheduledEventsTableRow.tsx

```diff
@@ -12,7 +17,52 @@ export interface ScheduledEventsTableRowProps {
   start_date: string;
 }
 
+interface RunInstancePayload {
+  start_at: Moment;
+  status: string;
+}
+
+const BASEPATH = api.routes.VIEW_SCHEDULED_EVENTS;
+
+const createRunInstance = async (
+  scheduleId: number | undefined,
+  data: RunInstancePayload,
+): Promise<any> => {
+  const token = await localForage.getItem<string>(localForageKeys.API_KEY);
+
+  return await axios.request({
+    url: `${BASEPATH}${scheduleId}${api.routes.CREATE_SCHEDULED_INSTANCE}`,
+    method: 'post',
+    headers: {
+      'Content-Type': 'application/json',
+      Authorization: `token ${token}`,
+    },
+    data,
+  });
+};
+
 export const ScheduledEventsTableRow: FunctionComponent<ScheduledEventsTableRowProps> = (props) => {
+  const [isLoading, setLoading] = useState(false);
+  const [scheduleId, setScheduleId] = useState<number>();
+
+  useEffect(() => {
+    setScheduleId(props.uid);
+  }, []);
+
+  const handleClick = () => {
+    setLoading(true);
+    createRunInstance(scheduleId, {
+      start_at: moment(),
+      status: 'p',
+    })
+      .then(() => {
+        setLoading(false);
+      })
+      .catch(() => {
+        setLoading(false);
+      });
```

Comment:
I'd rather you add a `console.log` of whatever went wrong.

Reply:
Now logs with a JSON string